packages/core/src/event-listeners/access-token.ts

```diff
@@ -0,0 +1,24 @@
+import { generateStandardId } from '@logto/shared';
+
+import { getUtcStartOfToday } from '#src/oidc/utils.js';
+import type Queries from '#src/tenants/Queries.js';
+
+export const accessTokenIssuedListener = async (
```

Comment:
not only for access tokens but for all token exchanges?

Reply:
Per discussion, "active user" is triggered by only "access token", resolve this.